server/routes/universities.js

```diff
@@ -6,45 +6,63 @@ const University = require('../models/universities');
 
 //GET all universities with populated courses
 router.get("/", async function(req, res, next) {
+
     const allUniversities = await University.find()
     .populate({path: 'courses', model: 'Course'})
     .catch(() => {
-        return res.status(500).send();
+        return next(new GeneralError('Error connecting to database'));
     });
-    res.status(200).send(allUniversities);
+    res.send(allUniversities);
 });
 
 //GET university with universityId and populated courses
 router.get('/:universityId/courses', async function(req, res, next) {
```

Comment:
Any reason for adding /courses, the route seems to work just with universityId?

Reply:
I don't think I added it. It might have been on dev when I merged it in to this branch? I'm not sure. But I'll change it back. Good catch